server/src/main/java/io/crate/metadata/sys/SysSegmentsTableInfo.java

```diff
@@ -62,6 +62,8 @@ public static SystemTable<ShardSegment> create(Supplier<DiscoveryNode> localNode
             .add("search", BOOLEAN, r -> r.getSegment().search)
             .add("version", STRING, r -> r.getSegment().getVersion().toString())
             .add("compound", BOOLEAN, r -> r.getSegment().compound)
+            .add("merge_id", STRING, r -> r.getSegment().getMergeId())
+            .add("fully_merged_docs", INTEGER, r -> r.getSegment().docsWithSource)
```

Comment:
Shouldn't `fully_merged_docs` show  `total docs - docs-with-source` ?

Reply:
Oops, yes it should.